tests/unit/cluster/manual-takeover.tcl

```diff
@@ -22,8 +22,12 @@ set paused_pid [srv 0 pid]
 set paused_pid1 [srv -1 pid]
 set paused_pid2 [srv -2 pid]
 test "Killing majority of master nodes" {
+    # Bumping the epochs to increase the chance of conflicts.
```

Comment:
without the fix, this can lead to increased conflicts, which can cause tests to fail

Reply:
i am fail to see why disbale the replica failover will work. Do you mean disable the it on the replica side? The code already doing it. Or do you mean disable it on the primary side?
```
# For this test, disable replica failover until
# all of the primaries are confirmed killed. Otherwise
# there might be enough time to elect a replica.
set replica_ids { 5 6 7 }
foreach id $replica_ids {
    R $id config set cluster-replica-no-failover yes
}

set paused_pid [srv 0 pid]
set paused_pid1 [srv -1 pid]
set paused_pid2 [srv -2 pid]
test "Killing majority of master nodes" {
    pause_process $paused_pid
    pause_process $paused_pid1
    pause_process $paused_pid2
}

foreach id $replica_ids {
    R $id config set cluster-replica-no-failover no
}
```

yes, it is a good idea to split it, i will find some ways to do it.

Reply:
>i am fail to see why disbale the replica failover will work. 

I think there is a race condition between turning off failover on the replicas and stopping the primary. I meant to suggest the following changes (reverting the order `R $id config set cluster-replica-no-failover no` and `pause_process $paused_pid`). Can you give it a try?

```suggestion
    foreach id $replica_ids {
        R $id config set cluster-replica-no-failover no
    }
    
    test "Killing majority of master nodes" {
        pause_process $paused_pid
        pause_process $paused_pid1
        pause_process $paused_pid2
   }
```

Reply:
What is the specific race? look like you are suggested to remove this one? e81bd15e992362062e2d60b614407667431cf248